kinta-lib/src/main/kotlin/com/dailymotion/kinta/integration/github/internal/GithubInit.kt

```diff
@@ -0,0 +1,24 @@
+package com.dailymotion.kinta.integration.github.internal
+
+import com.dailymotion.kinta.KintaConfig
+import com.dailymotion.kinta.KintaEnv
+import com.dailymotion.kinta.helper.CommandUtil
+import com.github.ajalt.clikt.core.CliktCommand
+
+
+object GithubInit : CliktCommand(name = "initGithub", help = "Set up Github to use it with Kinta") {
+
+    override fun run() {
+        val clientId = CommandUtil.prompt(message = "Provide your GitHub app client id :")
+        if (clientId?.isNotBlank() == true) {
+            KintaConfig.put(KintaEnv.GITHUB_APP_CLIENT_ID, clientId)
+            println("GITHUB_APP_CLIENT_ID has been set to your kinta.properties")
+
+            val clientSecret = CommandUtil.prompt(message = "Provide your Gitlab app client secret :")
+            if (clientSecret?.isNotBlank() == true) {
+                KintaConfig.put(KintaEnv.GITHUB_APP_CLIENT_SECRET, clientSecret)
+                println("GITHUB_APP_CLIENT_SECRET has been set to your kinta.properties")
```

Comment:
I'm not sure we should ask users to create an OAuth app. Either we can do it for them or we do like hub is doing and ask for username/password https://hub.github.com/hub.1.html